widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java

```diff
@@ -402,6 +410,7 @@ public void onDestroy(boolean retain) {
 
             getEngagementUseCase.unregisterListener(this);
             engagementEndUseCase.unregisterListener(this);
+            surveyUseCase.unregisterListener(this);
```

Comment:
move to onpause -
in current state when callview will be on top of Chat You will receive events to both views

Reply:
Looks like I need rebase with the develop branch.